src/config/intro.rst

```diff
@@ -157,7 +168,7 @@ You should be careful with changing configuration via the HTTP API since it's
 easy to make CouchDB unavailable. For instance, if you'd like to change the
 :option:`httpd/bind_address` for a new one::
 
-    curl -X PUT http://localhost:5984/_config/httpd/bind_address -d '"10.10.0.128"'
+    curl -X PUT http://localhost:5984/_node/node1/_config/httpd/bind_address -d '"10.10.0.128"'
```

Comment:
`node1@example.com`, thought we need to clarify where to get these node names (reference to `/_membership`) - they aren't arbitrary FQDN's.